vmware/docker-volume-vsphere
# Test to delete vm from its vmgroup and few dependent utils

This PR covers following things:
- Added a test to delete vm from its vmgroup - PR #929 
- Added govc utils to create and destroy vm.
- Made GetRandomNumber() public so it can be directly used. Eg. Get a unique name for vm

-------------------------------

Test steps:
1. Create a vm VM1 using govc
2. Create a vmgroup and associate VM1 to it.
3. Execute vmgroup ls command.
4. Delete the vm that was added to the vmgroup.
5. Again execute vmgroup ls command to verify command works fine.

Tested it locally. All test test-esx passed
```
=> Running target test-e2e-runalways
OK: 6 passed
--- PASS: Test (115.64s)
PASS

=> Running target test-e2e-runonce
--- PASS: Test (493.81s)
PASS
OK: 22 passed, 3 skipped
```
